### ISSUE TITLE
input bar needs reload to open links targeted at new Windows

### ISSUE THREAD

**Issue by tusch001** [scoute-dich/browser#543]
Sorry for reopening this.

Links on a page that are targeted at a new window open in a new tab but the page is white.
This is also true for links that are opened from other Apps.

Steps to reproduce on my phone with Android 6.0.1 Stock:

Use [startpage (de)](https://startpage.com/de) as search engine, make a search and open a link in the search results. JavaScript is generally disabled on my phone so I guess there is a target="blank" attribute in the link tag. The new tab opens and I can see the URI in the input bar. The tab stays white. Page content is only visible after reloading over the three-dot-menu.

This behaviour is not new and was also happening in the previous versions. It just took me some time to realise it wasn't fixed in 7.5 because I used a different search engine.

Maybe related also to #492? You asked for help there. How can I help / what do you need? There is nothing in the logcat as far as I remember.

**Comment by scoute-dich** [comment]
I think it's the Android version. Perhaps you can test the following apk-file:
[app-debug.apk.zip](https://github.com/scoute-dich/browser/files/6725087/app-debug.apk.zip)

**Comment by tusch001** [comment]
Wow. Fixed it.

I can open links in new taps from startpage. And external links from other Apps work, too. 

---

I also generated some bookmarks to see if #492 is working for me. I can open them in foreground and background. But I can't be sure because I save my bookmarks in the Nextcloud Bookmark App. 

I went back to the original version 7.5. There I can also open tabs from bookmarks. So no end result for that issue on my side. 

Thanks for this fix!
